src/systems/systems/src/shared/mod.rs

```diff
@@ -323,6 +323,18 @@ impl DelayedTrueLogicGate {
         }
     }
 
+    pub fn new_with_init_state(delay: Duration, initial_state: bool) -> DelayedTrueLogicGate {
```

Comment:
It is necessary to properly initialize this type of gate when in flight.

Let's remember how the gate works (with delay of 4 s):
- t: 0, in: false, out: false
- t: 1, in: true, out: false
- t: 2, in: true, out: false
[...]
- t: 5, in: true, out: true

Now if we spawn in flight, the simulation starts, and the value could be true right from the start. But the gate would always need the delay time to get true. Now this is a generic module, we don't know how everyone is using it. The delay could be small but in theory it could also be 5 minutes. Depending on the usage, there needs to be a way to initialize this gate in a fully true state from the start.

Reply:
OR: I just totally misunderstood the use of this :D   Let me have another look I'm tired :p

Reply:
The idea is that you can fundamentally initialize the gate in a true/true state. Once it goes to false shortly, the normal behavior should happen.

Reply:
Ok! So would it make sense to rename this to something like bypass_initial_delay ?

My idea behind this is this way the value you give there will not depend anymore on the type of gate, as a DelayedFalse you would still ask to "bypass initial delay = true"

Reply:
Yes...David made a good proposal for an improvement that I will use. By doing this, it gets much better understandable.

Reply:
Nice :)